### PR TITLE
Run the plone.app.event tests to see if our dev version causes problems

### PR DESCRIPTION
**Pull request by mauritsvanrees**
See https://github.com/collective/icalendar/issues/446

### CODE REVIEW

**Comment by mauritsvanrees** [comment]
Tests are green.

Could someone double check if this is good from the Plone side? Maybe @ale-rt or @gforcada ? Note that I already have a PR to [remove `icalendar` from the Plone core dev tests](https://github.com/plone/buildout.coredev/pull/817).

**Review by niccokunzmann — APPROVED** [review]

**Comment by niccokunzmann** [comment]
@mauritsvanrees, it is fine to merge with me. We can revert it or improve it once you have feedback from the Plone community.
Also: I wonder: Is the master there ever red? What is would be the policy if these tests are red? Do not ship & do not merge?

**Comment by mauritsvanrees** [comment]
I think the plone.app.event tests that we are running here are quite stable. If you see it go red when it is highly unlikely to be a problem in icalendar, you can ping me.

I would say: if the check here is red, and you merge, then the checks on master will be red. So best not merge. But at least I would say, do not release then.

**Pull request merged**
